.github/workflows/coverage.yml

```diff
@@ -0,0 +1,57 @@
+name: Code Coverage
+
+on: [ pull_request ]
```

Comment:
@nuxzero @ajzkk @aashishgurung I noticed that some of our other public libraries trigger this action on push:

https://github.com/omise/omise-woocommerce/blob/develop/.github/workflows/code-coverage.yml#L3-L4

What should be the trigger?

Reply:
Good point, I used `pull_request` to keep it same as the test workflow.

Reply:
I would say `push` because we need to run this action when something is pushed.

Reply:
@danfowler @aashishgurung I updated the trigger to `push`.